layouts/partials/head-css.html

```diff
@@ -12,3 +12,7 @@
 <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.css" />
 <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.10/styles/monokai.min.css">
 <link rel="stylesheet" href='{{ "css/style.css" | absURL }}' />
+{{ with .Site.Params.prism_syntax_highlighting }}
```

Comment:
You can remove this, as this is already included in the docsy folder.

Reply:
the prism css file also needs to be referenced but now from our static css folder similar to style.css.